src/main/webapp/i18n/de/iris.json

```diff
@@ -62,7 +67,7 @@
                     "breadcrumb": "Iris Einstellungen",
                     "global": "Globale Iris Einstellungen",
                     "course": "Kurs Iris Einstellungen",
-                    "programmingExercise": "Programmieraufgabe Iris Einstellungen"
+                    "exercise": "Programmieraufgabe Iris Einstellungen"
```

Comment:
```suggestion
                    "exercise": "Aufgabe Iris Einstellungen"
```

Reply:
This is actually a temporary fix to make the language files consistent. The client code expects this translation to be under "exercise". But it should be corrected in a later PR with all the rest of the renaming of the programming exercise chat feature (currently just called "exercise chat" across the codebase)